src/Controller/Simulation.java

```diff
@@ -511,7 +511,14 @@ public String toggleRunning() {
     private ArrayList<User> usersFromJSON(File srcFile) throws org.json.JSONException, IOException {
         if (srcFile == null) return null;
 
-        String wholeFile = new String(Files.readAllBytes(srcFile.toPath()));
+        String wholeFile;
+        if (srcFile.exists()) {
+            wholeFile = new String(Files.readAllBytes(srcFile.toPath()));
+        }
+        else {
+            wholeFile = "{\"Admin\":{\"password\":\"123456\",\"location\":null,\"type\":\"parent\"}}";
```

Comment:
I was thinking we generate a default file but this works too.  If we generated it then it would get added to the repo ig.